tests/integration/constraints/test_tabular.py

```diff
@@ -1,222 +0,0 @@
-        _is_valid, _bad_transform, _reverse_transform
-    )('col')
-    data = pd.DataFrame({'col': np.random.randint(1, 10, size=100)})
-    gc = GaussianCopula(constraints=[custom_constraint])
-    gc.fit(data)
-
-    # Run
-    sampled = gc.sample(100)
-
-    # Assert
-    assert all(sampled > 0)
-
-
-def test_fixed_increments():
-    """Test the ``FixedIncrements`` constraint end to end."""
-    # Setup
-    values = np.random.randint(1, 10, size=20) * 5
-    data = pd.DataFrame({'column': values})
-    constraint = FixedIncrements(column_name='column', increment_value=5)
-    gc = GaussianCopula(constraints=[constraint])
-    gc.fit(data)
-
-    # Run
-    sampled = gc.sample(100)
-
-    # Assert
-    assert all(sampled % 5 == 0)
-
-
-def test_inequality():
-    """Test the ``Inequality`` constraint end to end."""
-    # Setup
-    data = pd.DataFrame({
-        'low': np.random.randint(1, 10, size=20),
-        'high': np.random.randint(10, 20, size=20)
-    })
-    constraint = Inequality('low', 'high')
-    gc = GaussianCopula(constraints=[constraint])
-    gc.fit(data)
-
-    # Run
-    sampled = gc.sample(10)
-
-    # Assert
-    assert all(sampled['low'] <= sampled['high'])
-
-
-def test_scalar_inequality():
-    """Test the ``ScalarInequality`` constraint end to end."""
-    # Setup
-    data = pd.DataFrame({
-        'low': np.random.randint(1, 10, size=20),
-    })
-    constraint = ScalarInequality(column_name='low', value=11, relation='<')
-    gc = GaussianCopula(constraints=[constraint])
-    gc.fit(data)
-
-    # Run
-    sampled = gc.sample(10)
-
-    # Assert
-    assert all(sampled['low'] < 11)
-
-
-def test_range():
-    """Test the ``Range`` constraint end to end."""
-    # Setup
-    data = pd.DataFrame({
-        'low_column': np.random.randint(1, 5, size=20),
-        'middle_column': np.random.randint(6, 10, size=20),
-        'high_column': np.random.randint(11, 20, size=20),
-    })
-    constraint = Range(
-        low_column_name='low_column',
-        middle_column_name='middle_column',
-        high_column_name='high_column',
-        strict_boundaries=True
-    )
-
-    gc = GaussianCopula(constraints=[constraint])
-    gc.fit(data)
-
-    # Run
-    sampled = gc.sample(100)
-
-    # Assert
-    assert sampled.middle_column.min() >= sampled.low_column.min()
-    assert sampled.middle_column.max() <= sampled.high_column.max()
-
-
-def test_scalar_range():
-    """Test the ``ScalarRange`` constraint end to end."""
-    # Setup
-    data = pd.DataFrame({
-        'column': np.random.randint(6, 10, size=20),
-    })
-    constraint = ScalarRange(
-        column_name='column',
-        low_value=5,
-        high_value=11,
-        strict_boundaries=True
-    )
-
-    gc = GaussianCopula(constraints=[constraint])
-    gc.fit(data)
-
-    # Run
-    sampled = gc.sample(100)
-
-    # Assert
-    assert sampled.column.min() >= 5
-    assert sampled.column.max() <= 11
```

Comment:
we can probably add all these in a separate PR like you mentioned and just call the `validate` method on the sampled data to make sure the constraints are valid